src/main/scala/temple/test/internal/CRUDServiceTest.scala

```diff
@@ -16,6 +20,56 @@ class CRUDServiceTest(
   usesAuth: Boolean,
 ) extends ServiceTest(serviceName, baseURL, usesAuth) {
 
+  private def serviceReadable: Metadata.Readable =
+    service.lookupLocalMetadata[Metadata.Readable].getOrElse {
+      if (usesAuth) Metadata.Readable.This else Metadata.Readable.All
```

Comment:
doesn’t ProjectConfig have something for this?

Reply:
Ah so it does! Thanks